package io.github.fernandosilva94.cadastro.exceptions;

public class InativarUsuarioException extends RuntimeException {
    public InativarUsuarioException(String message, Throwable cause) {
        super(message, cause);
    }
}
